bills/models.py

```diff
@@ -16,6 +18,13 @@ class Participant(models.Model):
         on_delete=models.CASCADE,
         related_name="participants",
     )
+    user = models.ForeignKey(
```

Comment:
It is the event that should be assigned to user, not the Participant

Reply:
Done